### PR TITLE
feat(bigquery): Add example for creating a tag key

### PR DESCRIPTION
**Pull request by pratikmore0290**
## Description

Fixes #353964666

Note: If you are not associated with Google, open an issue for discussion before submitting a pull request.

## Checklist

**Readiness**

- [x] Yes, **merge** this PR after it is approved
- [ ] No, don't **merge** this PR after it is approved

**Style**

- [x] My sample follows the rules described for Terraform in the [Effective Samples style guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [x] My sample follows the other requirements and best practices in the [Contributing
guide](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#other-requirements-and-best-practices)

**Testing**

- [x] I have performed tests described in the [Contributing guide](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md):

   - [x] **[Tests](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#set-up-the-test-environment)** pass: `terraform apply`
   - [x] **[Lint](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#linting-and-formatting)** pass: `terraform fmt` check

**Intended location**

- [x] Yes, this sample will be (or already is) included on cloud.google.com
      Location(s): https://cloud.google.com/resource-manager/docs/tags/tags-creating-and-managing#creating_tag

- [ ] No, this sample won't be included on cloud.google.com
      Reason:

**API enablement**

- [ ] If the sample needs an API enabled to pass testing, I have added the service to the [Test setup file](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/test/setup/main.tf)

**Review**

- [ ] If this sample adds a new directory, I have added codeowners to the [CODEOWNERS file](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/.github/CODEOWNERS)

### CODE REVIEW

File: bigquery/bigquery_tags_tag_key/main.tf [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+# [START bigquery_tags_tag_key]
+resource "google_tags_tag_key" "env_tag_key" {
+  parent     = "projects/myproject1-381000"
```
Comment (wj-chen) [inline bigquery/bigquery_tags_tag_key/main.tf:19]: Can we reference project like this instead: https://github.com/terraform-google-modules/terraform-docs-samples/blob/3c210c5a52bba6381da35dc2ba75858ae7d3d190/gke/autopilot/tag/main.tf#L32

Comment (pratikmore0290) [reply bigquery/bigquery_tags_tag_key/main.tf]: Done! Thanks!

**Review by wj-chen — APPROVED** [review]

**Review by glasnt — APPROVED** [review]

**Pull request merged**